src/Paprika.Tests/Store/AbandonedTests.cs

```diff
@@ -250,4 +243,61 @@ public async Task Reuse_in_grow_and_shrink()
             await block.Commit(CommitOptions.FlushDataAndRoot);
         }
     }
+
+    [Test]
+    [Category(Categories.LongRunning)]
+    public async Task Abandoned_chain_creation_with_overflow()
+    {
+        // Minimum iterations required to overflow the abandoned list (AbandonedList.MaxCount).
+        const int minIterations = 60;
+
+        // Minimum accounts required to overflow an abandoned page (AbandonedPage.Payload.MaxCount * 2).
+        const int numAccounts = 2000;
+
+        var keccaks = Initialize(numAccounts);
+
+        // Set big value.
+        var accountValue = new byte[2900];
+        new Random(17).NextBytes(accountValue);
+
+        using var db = PagedDb.NativeMemoryDb(140000 * Page.PageSize, HistoryDepth);
```

Comment:
Less than 1GiB if I get this right. Is it heavy to have it runnable on CI and remove the `LongRunning` maybe? `LongRunning` are not run on CI atm.